src/uk/gov/defra/ffc/DefraUtils.groovy

```diff
@@ -94,6 +94,80 @@ def provisionInfrastructure(target, item, parameters) {
   }
 }
 
+def __getRbacGroups() {
+  return [ 'developer', 'administrator' ]
+}
```

Comment:
This could be replaced with a top level scoped def with the appropriate annotation, i.e.
```
@groovy.transform.Field
def rbacGroups = 
[ 
  'developer', 
  'administrator'
]
```

The field could then be used directly in a few functions, reducing the parameters being passed around, i.e.

```
def __setupRbacForNamespace(namespace) {
  for (group in rbacGroups) {
  ...
```

**Note:** the other top level defs should have these annotations

Reply:
I've refactored to use @Field, removed some duplication in the process and used a map `each` closure which seems more elgant than `for` in this scenario.

None of the other top level defs are used I believe bit this will be addressed in #27